src/textual/css/model.py

```diff
@@ -12,21 +12,27 @@
 from .types import Specificity3
 
 if TYPE_CHECKING:
+    from typing_extensions import Self
+
     from ..dom import DOMNode
 
 
 class SelectorType(Enum):
-    UNIVERSAL = 1
-    TYPE = 2
-    CLASS = 3
-    ID = 4
-    NESTED = 5
+    """Type of selector."""
+
+    UNIVERSAL = 1  # i.e. *
```

Comment:
Small suggestion rather than anything else: perhaps these comments should be docstrings for each of the values; then anyone working *on* Textual will get the hint-text benefits from their LSP?